zebra-chain/src/sapling/commitment.rs

```diff
@@ -137,6 +137,15 @@ impl NoteCommitment {
 #[derive(Clone, Copy, Deserialize, PartialEq, Serialize)]
 pub struct ValueCommitment(#[serde(with = "serde_helpers::AffinePoint")] pub jubjub::AffinePoint);
 
+impl<'a> std::ops::Add<&'a ValueCommitment> for ValueCommitment {
+    type Output = Self;
+
+    fn add(self, rhs: &'a ValueCommitment) -> Self::Output {
+        let value = self.0.to_extended() + rhs.0.to_extended();
+        ValueCommitment(value.into())
+    }
+}
+
```

Comment:
can this be implemented in terms of the existing Add impl?
```suggestion
impl<'a> std::ops::Add<&'a ValueCommitment> for ValueCommitment {
    type Output = Self;

    fn add(self, rhs: &'a ValueCommitment) -> Self::Output {
        self + *rhs
    }
}

```